src/Illuminate/Support/ViewErrorBag.php

```diff
@@ -33,7 +33,7 @@ public function hasBag($key = 'default')
      */
     public function getBag($key)
     {
-        return Arr::get($this->bags, $key, new MessageBag);
+        return Arr::get($this->bags, $key) ?: new MessageBag;
```

Comment:
In the first case object is created everytime, while in second - only if value in bags is not present.